src/common/components/repositories-list/index.js

```diff
@@ -59,17 +62,34 @@ class RepositoriesList extends Component {
     }
   }
 
+  onPageLinkClick(pageNumber) {
+    this.props.dispatch(fetchUserRepositories(pageNumber));
+  }
+
   render() {
     const isLoading = this.props.repositories.isFetching;
+    const pageLinks = this.props.repositories.pageLinks;
 
     return (
       <div>
+        { this.props.repositories.success && pageLinks &&
+          <div className={ styles['page-links-container'] }>
+            Pages: <PageLinks { ...pageLinks } onClick={ this.onPageLinkClick.bind(this) } />
```

Comment:
The "Pages:" text seems a bit unnecessary, here and below; I'd say the links should probably stand by themselves.